src/blocks/share/test/share.cypress.js

```diff
@@ -88,6 +89,7 @@ describe( 'Test CoBlocks Share Block', function() {
 	/**
 	 * Test the coblocks share block button size.
 	 */
+	// Weird CSS behaviour : real width does not match CSS width
```

Comment:
Feel free to remove the comment if you are not able to reproduce the failing test.

Reply:
I left a few comments in places where we might encounter these errors again.